From 6be3807187750cf1c0795ef671da5bd8a7f27117 Mon Sep 17 00:00:00 2001
From: Hongzhang Yang <Hongzhang.Yang@freescale.com>
Date: Mon, 16 Apr 2012 19:33:39 +0800
Subject: [PATCH 201/280] ENGR00179726-2 Add VDOA V4L support to VPU unit test

Tiled formats (IPU_PIX_FMT_TILED_NV12/IPU_PIX_FMT_TILED_NV12F) added since VPU
unit test will reference "ipu.h"

Signed-off-by: Hongzhang Yang <Hongzhang.Yang@freescale.com>
---
 ipu/ipu.h | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/ipu/ipu.h b/ipu/ipu.h
index 71a10f7..278385c 100644
--- a/ipu/ipu.h
+++ b/ipu/ipu.h
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2011 Freescale Semiconductor, Inc.
+ * Copyright 2005-2012 Freescale Semiconductor, Inc.
  */
 
 /*
@@ -129,6 +129,10 @@ typedef enum {
 #define IPU_PIX_FMT_VYU444  fourcc('V', '4', '4', '4')	/*!< 24 VYU 4:4:4 */
 /* two planes -- one Y, one Cb + Cr interleaved  */
 #define IPU_PIX_FMT_NV12    fourcc('N', 'V', '1', '2') /* 12  Y/CbCr 4:2:0  */
+/* two planes -- 12  tiled Y/CbCr 4:2:0  */
+#define IPU_PIX_FMT_TILED_NV12    fourcc('T', 'N', 'V', 'P')
+#define IPU_PIX_FMT_TILED_NV12F   fourcc('T', 'N', 'V', 'F')
+
 /*! @} */
 /*! @name YUV Planar Formats */
 /*! @{ */
-- 
1.8.0

